packer/ubuntu-14.04_amd64-droplet.json

```diff
@@ -56,11 +53,11 @@
     }
   ],
   "push": {
-    "name": "capgemini/apollo-mesos-ubuntu-14-04-amd64"
+    "name": "Enxebre/apollo-mesos-ubuntu-14-04-amd64"
```

Comment:
As we're changing the base images here (different from 0.1 release) we're going to have to figure out how to version the vagrant box and the AMIs correctly in this pull request